src/lex.js

```diff
@@ -114,10 +114,17 @@ window.Firth.lex = (function (Firth) {
                         break;
                     }
                 }
-                tokens.push({
-                    type: 'variable',
-                    name: token
-                });
+                if (token === 'true' || token === 'false') {
```

Comment:
Relevant: http://python-history.blogspot.co.uk/2013/11/story-of-none-true-false.html

No harm in making them literals. We can always change it later, of course!